### PR TITLE
Convert softmax lambda to keras layer

### PR DESCRIPTION
**Pull request by doronHarnoy**
In this PR we are converting Lambda layer representing softmax layer in Keras Softmax layer

### CODE REVIEW

**Comment by yotamazriel** [comment]
How tested this?

**Comment by doronHarnoy** [comment]
I'm using the network I got from Asaf removing all lambdas, investigating the model to work, and trying to import to our system

**Comment by yotamazriel** [comment]
Let's try to think about tests we can use here

**Comment by doronHarnoy** [comment]
I Agree, looks like the repo has a couple of tests using common networks, ill start with them and then lets see

**Review by arikmaor — COMMENTED** [review]
If tests run fine
I think you should create a PR to the original author as well
If he will review the PR it would provide very nice for us

**Review by yotamazriel — APPROVED** [review]
LGTM
